docs/using_swagger.rst

```diff
@@ -16,25 +16,25 @@ To make them accessable to `swagger-ui` they must be placed onto an webserver.
 
 .. code-block:: bash
 
-    php swagger.phar /projects/my_project  -o /var/html/swagger-docs
+    php vendor/zircote/swagger-php/bin/swagger /projects/my_project  -o /var/html/swagger-docs
 
 Check the help for additional options.
 
 .. code-block:: bash
 
-    php swagger.phar --help
+    php vendor/zircote/swagger-php/bin/swagger --help
```

Comment:
This is generally accessible `php vendor/bin/swagger` unless the user has modified the bin install path themselves in their composer.json.